draft-ietf-tls-rfc8447bis.md

```diff
@@ -535,6 +535,19 @@ IANA is requested to add a "Comment" column to the following registries:
 This list of registries is all registries that do not already have a
 "Comment" or "Notes" column or that were not orphaned by TLS 1.3.
 
+# Expert Review of Current and Potential IETF and IRTF Documents
```

Comment:
If this is supposed to be guidance to the experts, how are we planning to promulgate it?  Most of our current guidance to the experts is present as "note"s on the registry in question; do we expect this text or some similar text to be present in such a fashion?  If so, on which registries?

Reply:
I do not think it is.  I view the notes as content that is more intended for those reading/using the registry. In my experience the major source of guidance are RFC 8126 and 8447.

Reply:
I mean, I think that "designated experts should ..." reads like guidance to the experts.  Now, we could well say that we expect the experts to be familiar with the RFCs that are references for the registry as a whole, and opt to not put a note in the registry itself, but I would like that to be a conscious WG decision.

Reply:
@kaduk s17 of RFC 8447 includes instructions for DEs. Could we add a reference to that section to make to make it clear to all that the DEs will follow this?